multicluster/controllers/multicluster/common/types.go

```diff
@@ -30,8 +30,12 @@ const (
 	InvalidClusterID    = ClusterID("invalid")
 	InvalidClusterSetID = ClusterSetID("invalid")
 
-	DefaultWorkerCount = 5
+	DefaultWorkerCount       = 5
+	LabelIdentityWorkerCount = 50
```

Comment:
Maybe add a comment about why we need special setting for LabelIdentity worker

Reply:
Thanks. Added.